opencolorio_config_aces/clf/transforms/arri/generate.py

```diff
@@ -0,0 +1,445 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+"""
+*ARRI* CLF Transforms Generation
+=======================================
+
+Defines procedures for generating ARRI *Common LUT Format* (CLF)
+transforms:
+
+-   :func:`opencolorio_config_aces.clf.generate_clf_arri`
+"""
+
+from math import log, log10
+import PyOpenColorIO as ocio
+from pathlib import Path
+import sys
+
+from opencolorio_config_aces.clf.transforms import (
+    clf_basename,
+    format_clf_transform_id,
+    generate_clf_transform,
+    matrix_RGB_to_RGB_transform,
+)
+from opencolorio_config_aces.config import transform_factory
+
+__author__ = "OpenColorIO Contributors"
+__copyright__ = "Copyright Contributors to the OpenColorIO Project."
+__license__ = "New BSD License - https://opensource.org/licenses/BSD-3-Clause"
+__maintainer__ = "OpenColorIO Contributors"
+__email__ = "ocio-dev@lists.aswf.io"
+__status__ = "Production"
+
+__all__ = [
+    "FAMILY",
+    "GENUS",
+    "VERSION",
+    "generate_clf_arri",
+]
+
+FAMILY = "ARRI"
+"""
+*CLF* transforms family.
+"""
+
+GENUS = "Input"
+"""
+*CLF* transforms genus.
+"""
+
+VERSION = "1.0"
+"""
+*CLF* transforms version.
+"""
+
+
+def _build_awg3_mtx():
+    """
+    Build the `MatrixTransform` for ARRI Wide Gamut 3 primaries.
```

Comment:
ARRI --> Alexa  ??
(Same question on line 77.)

Reply:
"ARRI" is the latest prefix for these names. This is explained in the LogC4 White Paper.